<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnalisisWidget</class>
 <widget class="QWidget" name="AnalisisWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="LeftContainer">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame#LeftContainer {
	  border: 1px solid black; 
     /*border-color: rgb(135, 177, 255)*/;
     border-radius: /*5*/ 4px; 
	padding: 1px;
    background-color: rgb(235, 235, 235);
}

QWidget {
	background: transparent;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="containerFrame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame#containerFrame { 
     border: 1px solid rgb(135, 177, 255); 
     border-radius: 4px; 
	padding: 2px;
    background-color: rgb(235, 235, 235);
 } 

 QWidget {
	 background-color:  rgb(235, 235, 235);
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="header" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Filters</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="hideButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">QToolButton#hideButton
{
	border: none;
}</string>
              </property>
              <property name="text">
               <string>hide</string>
              </property>
              <property name="icon">
               <iconset resource="Icons.qrc">
                <normaloff>:/resources/icons/strzalka-dol.png</normaloff>
                <normalon>:/resources/icons/strzalka-gora.png</normalon>:/resources/icons/strzalka-dol.png</iconset>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QScrollArea" name="scrollArea">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>330</width>
             <height>160</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QFrame#scrollArea
{
	border:none;
}</string>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOn</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAsNeeded</enum>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="filterScroll">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>314</width>
              <height>263</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="treeHolder" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QTreeView::branch:has-children:!has-siblings:closed,
     QTreeView::branch:closed:has-children:has-siblings {
             border-image: none;
             image: url(:/resources/icons/trojkat-8x8-dodolu.png);
     }

     QTreeView::branch:open:has-children:!has-siblings,
     QTreeView::branch:open:has-children:has-siblings  {
             border-image: none;
             image: url(:/resources/icons/trojkat-8x8-naboku.png);
     }

#treeHolder
{
	background: white;
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="analisisArea" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget {
	background: transparent;
}

#analisisArea {
	background-color: rgb(91, 91, 91);
}

#analisisArea &gt; *, EDRDockWidgetSet &gt; * {
	border: 1px solid black;
	border-radius: 4px;
}

#analisisArea QMainWindow {
	background-color: white;
	border-bottom-left-radius: 0px;
	border-bottom-right-radius: 0px;
}

QMainWindow &gt; * {
	background-color: rgb(91, 91, 91);
}

EDRDockWidgetSet &gt; * {
	border-top: none;
	border-top-left-radius: 0px;
	border-top-right-radius: 0px;
}

QMainWindow EDRTitleBar, QMainWindow EDRTitleBar * {
	background-color: rgb(43, 42, 41);
	color: white;
	margin: 0px;
	border: none;
	border-radius: 0px;
}

QMainWindow EDRTitleBar {
	border-top-left-radius: 4px;
	border-top-right-radius: 4px;
}

QDockWidget &gt; .QWidget, EDRDockWidget &gt; .QWidget, VisualizerWidget &gt; .QWidget, EDRDockWidgetSet &gt; .QWidget {
	border: 1px solid black;
	border-top: none;
	border-top-left-radius: 0px;
	border-top-right-radius: 0px;
	border-bottom-left-radius: 4px;
	border-bottom-right-radius: 4px;
	background: white;
}

EDRDockWidgetSet &gt; EDRTitleBar {
	margin-bottom: -1px;
	border-bottom: 2px solid white;
	border-top-left-radius: 4px;
	border-top-right-radius: 4px;
	border-bottom-left-radius: 0px;
	border-bottom-right-radius: 0px;
}

QMainWindow::separator {
	background-color: transparent;
	width: 4px;
	height: 4px;
}

QMainWindow::separator:hover {
	background-color: rgb(190, 210, 250);
}

QMainWindow::separator:vertical {
	image: url(:/resources/icons/splitter-dots-vertical.png);
 }

QMainWindow::separator:horizontal {
	image: url(:/resources/icons/splitter-dots-horizontal.png);
}

QMainWindow::separator:vertical:hover {
	margin-top: 4px;
	margin-bottom: 4px;
 }

QMainWindow::separator:horizontal:hover {
	margin-left: 4px;
	margin-right: 4px;
}

/* previous
QMainWindow QTabBar::tab {
	border: 1px solid  rgb(91, 91, 91);
	border-radius: 0px;
	background-color: rgb(91, 91, 91);
	color: white;
	min-width: 8ex;
	padding: 2px;
 }

QMainWindow QTabBar::tab:hover {
     background-color: #898989;
 }

QMainWindow QTabBar::tab:selected {
	border: 1px solid  rgb(41, 41, 41);
	background-color: rgb(41, 41, 41);
	border-top-left-radius: 4px;
	border-top-right-radius: 4px;
 }
*/

/*
QWidget {}
*/

QTabBar, QTabBar * {}

QTabBar::tab {
	border: 1px solid  black;
	border-top: none;
	border-radius: 0px;
	background-color: transparent;
	color: white;
	min-width: 10ex;
	padding: 2px;
	margin: 0px;
}

QTabBar::tab:previous-selected {
	border-right: none;
}

QTabBar::tab:next-selected, QTabBar::tab:middle:next-selected {
	border-right: 1px solid black;
}

QTabBar::tab:first {
	border: 1px solid  black;
	border-top: none;
	border-bottom-left-radius: 4px;
}

QTabBar::tab:first:previous-selected {
	border-right: none;
}

QTabBar::tab:middle {
	border: 1px solid  black;
	border-top: none;
	border-left: none;
}

QTabBar::tab:last {
	border: 1px solid  black;
	border-top: none;
	border-left: none;
	border-bottom-right-radius: 4px;
}

QTabBar::tab:middle:selected {
	border: 1px solid  black;
	background-color: black;
	border-bottom-left-radius: 4px;
	border-bottom-right-radius: 4px;
	margin-left: -1px;
	margin-right: -1px;
}

QTabBar::tab:first:selected {
border: 1px solid black;
	background-color: black;
	border-bottom-left-radius: 4px;
	border-bottom-right-radius: 4px;
	margin-left: 0px;
	margin-right: -1px;
}

QTabBar::tab:last:selected {
border: 1px solid  black;
	background-color: black;
	border-bottom-left-radius: 4px;
	border-bottom-right-radius: 4px;
		margin-left: -1px;
	margin-right: 0px;
}

QTabBar::tab:first:!selected, QTabBar::tab:last:!selected, QTabBar::tab:middle:!selected {
	margin-bottom: 4px;
}

QTabBar::tab:hover {
    background-color: #898989;
}
</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections/>
</ui>
